.prettierignore

```diff
@@ -0,0 +1,3 @@
+# Ignore artifacts:
+build
+coverage
```

Comment:
Does this project generate these files?

Reply:
i am not sure! i poked around and couldn't tell, but i also wasn't sure how identity-site generates them, so i left this in to be safe. happy to remove it if i was just being overly cautious!

Reply:
sounds good! love to remove things!!!